lmfdb/hypergm/plot.py

```diff
@@ -47,7 +47,9 @@ def piecewise_linear_image(A,B):
     
 def circle_image(A,B):
     G = Graphics()
-    G += circle((0,0), 1 , color = 'grey')
+    G += circle((0,0), 1 , color = 'black',thickness = 3)
+    backgroundColor = (227/255,242/255,253/255) #This is the RGB color (in Sage format) of the properties bar (found in style.css). Making the framing circle this color makes it invisible on the page. 
```

Comment:
`(227/255,242/255,253/255, 0)` might work better and work independently of the current theme

Reply:
Good idea; I'll edit it in and test it out to make sure.

Reply:
Turns out there is a builtin transparency parameter ("alpha") for graphics that I missed. The frame should now be invisible, regardless of background theme.